lambdas/issuecredential/src/main/java/uk/gov/di/ipv/cri/kbv/api/service/EvidenceFactory.java

```diff
@@ -67,6 +69,14 @@ public Object[] create(KBVItem kbvItem) throws JsonProcessingException {
         return new Map[] {objectMapper.convertValue(evidence, Map.class)};
     }
 
+    private int getVerificationScoreFromSession(SessionItem sessionItem) {
+        if (Objects.isNull(sessionItem) || Objects.isNull(sessionItem.getEvidenceRequest())) {
+            return VC_PASS_EVIDENCE_SCORE;
+        }
+        int verificationScore = sessionItem.getEvidenceRequest().getVerificationScore();
+        return verificationScore > 0 ? verificationScore : VC_PASS_EVIDENCE_SCORE;
+    }
```

Comment:
You only need to null check the evidenceRequest, I would do it this way b'cos it looks like existing code here

```
private int getVerificationScore(EvidenceRequest evidenceRequest) {
    return Optional.ofNullable(evidenceRequest)
                   .map(EvidenceRequest::getVerificationScore)
                   .filter(score -> score > 0)
                   .orElse(VC_PASS_EVIDENCE_SCORE);
}

But this also very good
```
   private int getVerificationScore(EvidenceRequest evidenceRequest) {
        if (Objects.isNull(evidenceRequest)) {
            return VC_PASS_EVIDENCE_SCORE;
        }
        return verificationScore > 0 ? evidenceRequest.getVerificationScore() : VC_PASS_EVIDENCE_SCORE;
    }
```